docs/scaling/README.md

```diff
@@ -1,3 +1,21 @@
-# Knative Serving Scaling
-
-TODO: write developer/operator facing documentation.
+# Knative Serving Scaling
+
+TODO: write developer/operator facing documentation.
```

Comment:
You can delete the TODO (and all the old contents of this file).  Developer facing docs are here: https://github.com/knative/docs/tree/master/serving/samples/autoscale-go